samples/javascript_nodejs/81.skills-skilldialog/README.md

```diff
@@ -0,0 +1,87 @@
+# SkillDialog
+
+Bot Framework v4 Skills with Dialogs sample.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com); it shows how to use a skill dialog from a root bot.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org) version 10.14 or higher
```

Comment:
```suggestion
- [Node.js](https://nodejs.org) version 10.14 or higher.
```

Reply:
I agree, 80 and 81 should be very similar (or the same as much as we can). For the C# portion, I am waiting for Gary to merge the 3.1 updates and I will go back to the 80 sample to make sure it looks closer to what we have in 81.

Reply:
Yeah, I should've used Sample 80 more instead of pulling from Core Bot and then trying to port from C#. I'll make the error handlers match and see if I can spot other differences where it would make sense to fix.

Reply:
Error handlers match sample 80, now. I don't believe any other differences require an adjustment...they're mostly SkillDialog (sample 81) vs. non-SkillDialog (sample 80) differences.

I didn't change this sample's Skill's error handler since it matched sample 80 already (except this one will also clear conversationState and stringify's the error --**Should I make this change to sample 80 as well?**). I don't believe it needs to match the Root error handler since it doesn't send the endOfConversation the same way.

Reply:
Yes, make them look the same. Thanks

Reply:
https://github.com/microsoft/BotBuilder-Samples/pull/2253

Reply:
approved